Test name: StatisticalFunctionsTest::testStatisticalFormulasFODS
forced failure
- Testing file:///<<PKGBUILDDIR>>//sc/qa/unit/data/functions/statistical/fods/KahanSum.fods failed, Sheet2.A1 '' result: 0, expected: 0

Failures !!!
Run: 1   Failure total: 1   Failures: 1   Errors: 0
make[3]: *** [/<<PKGBUILDDIR>>/solenv/gbuild/CppunitTest.mk:130: /<<PKGBUILDDIR>>/workdir/CppunitTest/sc_statistical_functions_test.test] Error 1

diff --git a/sc/inc/kahan.hxx b/sc/inc/kahan.hxx
index ac97ae4394fa..f0d0ade29825 100644
--- a/sc/inc/kahan.hxx
+++ b/sc/inc/kahan.hxx
@@ -71,7 +71,7 @@ public:
       */
     inline void add(const KahanSum& fSum)
     {
-#ifdef _WIN32
+#if defined _WIN32 || defined __aarch64__
         // For some odd unknown reason WIN32 fails badly with the
         // sum+compensation value. Continue keeping the old though slightly off
         // (see tdf#156985) explicit addition of the compensation value.
@@ -89,7 +89,7 @@ public:
       */
     inline void subtract(const KahanSum& fSum)
     {
-#ifdef _WIN32
+#if defined _WIN32 || defined __aarch64__
         add(-fSum.m_fSum);
         add(-fSum.m_fError);
 #else
